src/components/Contributors.tsx

```diff
@@ -5,8 +5,7 @@ import { Box, Flex, Image, LinkBox, LinkOverlay } from "@chakra-ui/react"
 import InlineLink from "@/components/Link"
 import Text from "@/components/OldText"
 
-// TODO: Re-import once Crowdin scripts are functional and data available
-// import data from "../data/contributors.json"
+import data from "!!raw-loader!../../.all-contributorsrc"
```

Comment:
Can use shortcut imports here as well to avoid relative paths
```suggestion
import data from "!!raw-loader!@/../.all-contributorsrc"
```